### DISCUSSION TITLE
FullpageOS with Midori as default browser

### DISCUSSION THREAD

**Discussion by KroonCo** [guysoft/FullPageOS#581] [Ideas]
Thanks for the great work creating this distro. Seems like a lot of users felt a need for such a distro, including myself of course.
I always try to put old power efficient hardware to good use, so i still have a rpi model b+ and a rpi zero laying around. I was hoping i could tempt you to create a nightly build with midori as default browser, so al de convenient components that fullpageos gives can be use with less resources. is that something that can be done?

**Comment by KroonCo** [comment]
excuse me for not having a good look on what is already there

**Reply by guysoft** [reply]
Welcome to share if you manage to get it to work. I think it would still be too weak

**Reply by KroonCo** [reply]
I did manage to get it working, but loading any page that requires some resources (like our Zabbix dashboard needs memory) makes the browser (even Midori) break.
